scripts/release_scripts/run_macaron.sh

```diff
@@ -363,6 +363,10 @@ for v in "${proxy_var_names[@]}"; do
     [[ -n ${!v} ]] && proxy_vars+=("-e" "${v}=${!v}")
 done
 
+prod_var_names=(
+    "PYTHONWARNINGS=ignore"  # Turn off Python warnings in the production environment.
```

Comment:
We are missing the `-e` flag here. It's because we are passing it the runtime container through `docker run`